visualizer/style/parseTree.css

```diff
@@ -36,6 +43,10 @@
   background-color: #B5D5FF;
 }
 
+.input {
+  display: inline-block;
```

Comment:
If you end up creating a `#measuringDiv .input` rule like I suggested above, maybe this can go in there as well? (I'm assuming the measuring is the reason you had to add it here, because otherwise the `#expandedInput span` style should already do this.)